Swiftfin tvOS/Views/SettingsView/SettingsView.swift

```diff
@@ -51,14 +51,26 @@ struct SettingsView: View {
                     Button {
                         viewModel.signOut()
                     } label: {
-                        L10n.switchUser.text
-                            .foregroundColor(.jellyfinPurple)
+                        HStack {
+
+                            Text(L10n.switchUser)
+                                .foregroundColor(.jellyfinPurple)
+
+                            Spacer()
+
+                            Image(systemName: "chevron.right")
+                                .font(.body.weight(.regular))
+                                .foregroundColor(.secondary)
+                        }
                     }
+
+                } header: {
```

Comment:
Since you're here, please use the other init and you can do more throughout this file if you would like to.